.github/workflows/haskell.yml

```diff
@@ -62,8 +62,8 @@ jobs:
         cabal v2-run   --project-file=cabal.project.ci semantic-java:test
         cabal v2-run   --project-file=cabal.project.ci semantic-json:test
         cabal v2-run   --project-file=cabal.project.ci semantic-python:test
-        cabal v2-run   --project-file=cabal.project.ci semantic-python:test:compiling
-        cabal v2-run   --project-file=cabal.project.ci semantic-python:test:graphing
+        # cabal v2-run   --project-file=cabal.project.ci semantic-python:test:compiling
+        # cabal v2-run   --project-file=cabal.project.ci semantic-python:test:graphing
```

Comment:
We should consult with @joshvera and @BekaValentine about how/when we want to merge the efforts in this branch with the scope-graphing ones. We could make this branch based off of the stack graph one, or vice versa; either way, I’m happy to help with the plug-and-chug addition of `Success` branches.

Reply:
Happy to get this in and deal with the merge on the stack graph branch later.